### ISSUE TITLE
Microsoft Edge: Reading View notification is not announced

### ISSUE THREAD

**Issue by josephsl** [nvaccess/nvda#8423]
Hi,

Microsoft Edge comes with Reading view, a way to reformat the current page for smooth reading. However, NVDA does not announce certain notifications about it, including on pages where this view is not supported. Investigation shows this is tied to notification event introduced in IUIA5.

### Steps to reproduce:

1. Open Google with Edge.
2. Press Control+Shift+R to enable reading view.
> Please also remember to attach a zip of any files required to reproduce the issue.

### Expected behavior:
NVDA announces, "reading view is not available for this page".

### Actual behavior:
No announcement.

### System configuration:
NVDA next.15316 (installed), Windows 10 April 2018 Update.

### Other questions:
Reproducible: yes.
Did restart help: no.

### Technical details:
Certain notifications are sent via UiaRaiseNotificationEvent (see #7984 for details), and reading view announcement is one of them. At the moment NVDA checks to make sure focused control is part of the active application where the notification came from, but Edge uses at least two processes when run:

* MicrosoftEdge.exe: main process.
* MicrosoftEdgeCP.exe: content process.

Notifications are sent by the first process, thus failing the notification announcement criteria.

### Solution:
The Edge app module (microsoftedge.py) will gain notification event handler that'll announce all messages. This was tested on Windows 10 App Essentials add-on and was successful.

Thanks.
